src/Tizen.NUI/src/public/Common/StyleManager.cs

```diff
@@ -106,6 +117,35 @@ public static StyleManager Get()
             return ret;
         }
 
+        /// <summary>
+        /// Sets the brokenImageUrl.
+        /// the broken image is the image to show when image loading is failed.
+        /// When the broken image and type are set in the Application,
+        /// the proper brokenImage is set automatically considering the size of view and the size of the brokenImage.
+        /// This Api is used from theme manager.
+        /// </summary>
+        /// <param name="type"> The type for brokenImage </param>
+        /// <param name="url"> The url for brokenImage </param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void SetBrokenImageUrl(BrokenImageType type, string url)
```

Comment:
`StyleManager` is deprecate. Please change visibility of a new enum and methods to `internal` instead of `public`.